packages/contentstack-config/src/utils/interactive.ts

```diff
@@ -11,6 +11,7 @@ export const askRegions = async (): Promise<string> => {
       { name: 'EU', value: 'EU' },
       { name: 'AZURE-NA', value: 'AZURE-NA' },
       { name: 'AZURE-EU', value: 'AZURE-EU' },
+      // { name: 'GCP-NA', value: 'GCP-NA' },
```

Comment:
@cs-raj Can you remove this line rather commenting

Reply:
removed this too